src/main/steps/how-do-you-want-to-apply/template.njk

```diff
@@ -0,0 +1,30 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% extends "common/page.njk" %}
+
+{% block page_content %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+  <p class="govuk-body">{{ line1 }}</p>
+  <p class="govuk-heading-s">{{ subHeading1 }}</p>
+  <p class="govuk-body">{{ line2 }}</p>
+  <p class="govuk-heading-s">{{ subHeading2 }}</p>
+  <p class="govuk-body">{{ line3 }}</p>
+  <div class="govuk-inset-text">
+    <p class="govuk-body"> {{ line4 | safe }}</p>
+  </div>
+  <p class="govuk-body">{{ line5 }}</p>
+  {% set summary %}
+    <p>{{ helpText1 }}</p>
+    <p>{{ helpText2 }}</p>
+    <ul class="govuk-list govuk-list--bullet">
+      {% for item in helpPayingWhen %}
+        <li>{{ item | safe }}</li>
+      {% endfor %}
+    </ul>
+    <p>{{ helpText3 }}</p>
+  {% endset %}
+    {{ govukDetails({
+    summaryText: readMore,
+    html: summary
+  }) }}
```

Comment:
Could this be formatted more elegantly like this?
`{{ 
    govukDetails({
        summaryText: readMore,
        html: summary
    }) 
}}`
Just a thought...

Reply:
Was trying to put suggested formatting here but the comments code option doesn't seem to want to let me :(

Reply:
```
{{ 
    govukDetails({
        summaryText: readMore,
        html: summary
    })
}}
```